src/compute/src/render/mod.rs

```diff
@@ -247,7 +248,8 @@ pub fn build_compute_dataflow<A: Allocate>(
                     imported_sources.push((mz_expr::Id::Global(*source_id), (oks, errs)));
 
                     // Associate returned tokens with the source identifier.
-                    tokens.insert(*source_id, token);
+                    #[allow(clippy::as_conversions)]
```

Comment:
nit: instead of the allow, I do a `let token: Rc<dyn Any> = Rc::new(token)`. same number of lines and less cognitive overhead when reading the code